docs/kit/logistics/transport/packaging.md

```diff
@@ -0,0 +1,63 @@
+* The box should be sealed with cable ties and, if available high quality tape over the handles \(not selotape or insulating tape\). The Return Shipping Pack includes cables ties, but no tape \(as it's difficult to obtain a tiny roll of high quality tape\).
+
+## Return Shipping Pack
+
+When teams have to return a kit to us we provide them with a return shipping pack. This kit contains the following:
+
+| Item | Quantity | URL | Notes |
+| :--- | :--- | :--- | :--- |
+| 575x700mm polythene bag | 1 | [UK Packaging ref 08220](http://www.ukpackaging.com/postal-packaging/polythene-mailing-bags/grey-polythene-mailing-bags-575x700mm-60mu) | [Bag size research](../research.md#return-shipping-bag) |
+| 160x4.8mm cable tie | 5 | [Cableties Online](https://www.cabletiesonline.co.uk/cable-ties-b-w/cable-ties-160mm-x-4.html) | Two per handle. One spare. |
+| 112x80mm caution lithium ion battery label | 1 | [Limpet Labels](http://www.limpetlabels.co.uk/shop/view/293_Caution_Lithium_Battery_Labels/755_Caution_Lithium_Ion_Battery_Labels_%28112_x_80mm%29) |  |
+| A6 plain document wallet | 2 | [UK Packaging ref 10008](http://www.ukpackaging.com/document-wallets-a6-document-enclosed-wallets-plain) | One to use for address label. One spare. |
+| 750mm wide kraft paper | 2m | [UK Packaging ref 11042](http://www.ukpackaging.com/kraft-paper-rolls-imitation-kraft-kraft-paper-rolls-imitation-kraft-750mmx220m) |  |
+| 120x215mm \(internal\) gold jiffy bag | 4 | [UK Packaging ref 09033](http://www.ukpackaging.com/arofol-classic-postal-bags-gold-2) |  |
+| 170x245mm \(internal\) white jiffy bag | 4 | [UK Packaging ref 09312](http://www.ukpackaging.com/jiffy-earth-aware-airkraft-white-ak1-170x245mm-50-pack) |  |
+| 220x320mm \(internal\) white jiffy bag | 1 | [UK Packaging ref 09314](http://www.ukpackaging.com/postal-packaging/jiffy-bags/jiffy-earth-aware-airkraft-white-ak3-220x320mm-50-pack) | To put the whole return shipping pack into. |
+| Return shipping pack instructions | 1 | [Revision SR2017-2](https://github.com/srobo/return-shipping-pack/releases/download/SR2017-2/return-shipping-pack-instructions.pdf) |  |
+| Loan extension instructions | 1 | [Revision SR2017-2](https://github.com/srobo/return-shipping-pack/releases/download/SR2017-2/loanext-instructions.pdf) | Only included in return shipping packs handed out at the end of the competition. |
+| Kit list | 1 | [Kit Definition](https://bitbucket.org/richardbarlow/sr-kit-coord/wiki/Kit%20Definition) | Only included in return shipping packs handed out at the end of the competition. Remove 'disclaimer'  and the wire from the list. |
+
+### Instruction Notes
+
+1. Check all parts are present \(use list provided and notify LTC of any missing items\).
+2. Place parts into jiffy bags \(do not seal unless specified\):
+   1. Large white **\[photo of four bags with the parts in front\]:**
+      1. 1x tablet
```

Comment:
Worth noting that this is out of date from SR2019 onwards

Reply:
This is incorrect _for SR2019_. I think it's possible we'll go back to issuing tablets in some form (though I agree unlikely). Since there are plenty of other places which mention tablets and would need updating. I'm going to leave this for now.